src/components/widgets/toolhead/ToolheadMoves.vue

```diff
@@ -39,7 +39,7 @@
           small-icon
           @click="sendGcode('G28', waits.onHomeAll)"
         >
-          {{ $t('app.general.btn.all') }}
+          {{ $t('app.general.btn.home_all') }}
```

Comment:
shouldn't this go under `app.tool.btn.home_all`?

Reply:
hah, yep!